### DISCUSSION TITLE
java17分支 gateway的swagger问题

### DISCUSSION THREAD

**Discussion by stephen5538** [zuihou/zuihou-admin-cloud#299] [02.提问]
1. 使用java17分支 访问gateway的swagger时 其它服务接口都是404 not found，发现接口没有拼接其它服务前缀 
如下图没有拼接前缀authority，导致访问接口报错404 not found
![image](https://github.com/dromara/lamp-cloud/assets/25486527/9d6a2d95-eb1f-44da-a22c-0134076bc614)
![image](https://github.com/dromara/lamp-cloud/assets/25486527/dbec6b14-3ba6-479e-ba4f-be76b53decc4)

2.就是java17分支适应schema模式数据库时候是否需要将header中的tenant默认设置为0000 否则如果还是使用MDAwMA==的话直接会找不到这个数据库
![image](https://github.com/dromara/lamp-cloud/assets/25486527/e228ef09-e58b-42f0-b574-ac2d224a4914)

**Comment by zuihou** [comment]
1.  已知问题
2. 访问网关时，Tenant 传秘文； 直接访问服务，tenant 传明文： 0000。  因为网关会自动将密文转为明文
<img width="780" alt="image" src="https://github.com/dromara/lamp-cloud/assets/8500410/779ded2b-be45-48b6-8b6a-479bd85be2d8">

**Comment by zuihou** [comment]
问题1 已解决

https://github.com/dromara/lamp-cloud/commit/0f8053f0786ade970ae1f5b281bd3804a246f5b0
